src/util/dlist/dlist_debug.h

```diff
@@ -23,6 +23,8 @@
 #include <stdint.h>
 // #include <assert.h> /*it's required for assertion operation */
 
+#define ENABLE_DEBUG 1
```

Comment:
Avoid ambiguous macro names: this header can be included in some source files which relies on the macro with the same name. It's better to use something like `DLIST_DEBUG`.

Also, I guess this macro can be defined by some module option.

Reply:
Yep, gonna fix